app/assets/javascripts/webauthn.js

```diff
@@ -102,6 +102,48 @@
     });
   });
 
+  $(function() {
+    var sessionForm = $(".js-webauthn-session-cli--form");
+    var sessionSubmit = $(".js-webauthn-session-cli--submit");
+    var sessionError = $(".js-webauthn-session-cli--error");
+    var csrfToken = $("[name='csrf-token']").attr("content");
+
+    sessionForm.submit(function(event) {
+      var form = handleEvent(event);
+      var options = JSON.parse(form.dataset.options);
+      options.challenge = base64urlToBuffer(options.challenge);
+      options.allowCredentials = credentialsToBuffer(options.allowCredentials);
+      navigator.credentials.get({
```

Comment:
Very nostalgic to see the `navigator` here, given that it refers to a dead browser.

Reply:
lol is that where it comes from? I thought it made sense since the [user](https://developer.mozilla.org/en-US/docs/Web/API/Navigator) "navigates" to the given pages but I sincerely hope it's from the browser

Reply:
[Yep](https://www.oreilly.com/library/view/javascript-the-definitive/0596000480/ch13s06.html#:~:text=The%20Window.,also%20supported%20by%20Internet%20Explorer.)